101-bastion-hub-spoke-vnet/azuredeploy.json

```diff
@@ -0,0 +1,298 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "metadata": {
+                "description": "The location of all resources"
+            },
+            "defaultValue": "[resourceGroup().location]"
+        },
+        "vNetHubName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Hub vNet"
+            },
+            "defaultValue": "vnet-hub"
+        },
+        "vNetSpokeName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Spoke vNet"
+            },
+            "defaultValue": "vnet-spoke"
+        },
+        "vmName": {
+            "type": "string",
+            "metadata": {
+                "description": "The VM Name"
```

Comment:
this was ambiguous to me the first time through - couldn't tell if this was for the bastion host or the other vm, let's add a little more detail to the description